snap/snapcraft.yaml

```diff
@@ -0,0 +1,21 @@
+name: strictdoc
```

Comment:
Would it be possible to keep this `snap` folder not in the root folder but inside the `developer` folder (`developer/snap`) or this is required by snap's conventions?

Background: I would like to keep the root folder not too crowded if possible.

Reply:
Sounds reasonable. Fixed.